src/Options.js

```diff
@@ -487,6 +505,14 @@ export function addOptions(flags, commandOptions) {
     'sourceMaps generated to file or inline with data: URL',
     (to) => { return commandOptions.sourceMaps = to; }
   );
+  flags.option('--source-root <false|string|default>',
```

Comment:
In this PR currently:
`<string>` set the sourceRoot to this value,
`false` leave the sourceRoot undefined,
`default` set the sourceRoot to the basepath(output file).